src/utils.js

```diff
@@ -0,0 +1,40 @@
+function sendJsonResponse(res, statusCode, data) {
+  res.writeHead(statusCode, { 'Content-Type': 'application/json' });
+  res.end(JSON.stringify(data));
+}
+
+function respondWithError(res, errors) {
+  sendJsonResponse(res, 400, { errors });
+}
+
+function validateRequest(text, toCase, supportedCases) {
```

Comment:
Is `supportedCases` going to change anytime?
You could probably just store it in a variable in this file, before the function declaration, and access it

Reply:
@mfcastilho